omnisafe/utils/config_utils.py

```diff
@@ -60,6 +60,13 @@ def create_namedtuple_from_dict(obj):
         return obj
 
 
+def create_dict_from_namedtuple(obj):
+    """Create dict from name-tuple"""
+    if isinstance(obj, tuple):
```

Comment:
May need to check the `tuple` object is a `namedtuple`.

```suggestion
    if isinstance(obj, tuple) and hasattr(obj, '_fields'):
```